composer.json

```diff
@@ -15,7 +15,7 @@
     },
     "require": {
         "php": "^7.1 || ^8.0",
-        "php-di/php-di": "^6.0.0",
+        "php-di/php-di": "^7.0.0",
```

Comment:
I'm thinking we could support `^6.0|^7.0` if there are no downsides?

Reply:
I didn't find any. Should be fine!

Reply:
👍 I'll wait for that change before merging then.

Reply:
Done